src/content/docs/style-guide/writing-docs/article-templates/agent-api-guide-template.mdx

```diff
@@ -326,7 +326,7 @@ APM includes a number of ways to record arbitrary custom data. For an explanatio
   <tbody>
     <tr>
       <td>
-        Send data about an event so you can analyze it in New Relic Insights...
+        Send data about an event so you can analyze it when querying...
```

Comment:
I'm intrigued and baffled by the ellipses in this doc, but it's outside the scope of this PR, I think

Reply:
haha. yeah best left for another day